exasol/analytics/query/handler/query/drop/interface.py

```diff
@@ -0,0 +1,9 @@
+from exasol.analytics.query.handler.query.interface import Query
+
+
+class DropyQuery(Query):
+    pass
+
+
+class DropQuery(Query):
+    pass
```

Comment:
Duplicated?

Reply:
Haha -- the first class contains an extra `y`: Dop-y-Query :rofl: